book/website/project-design/data-governance/data-gov-ml.md

```diff
@@ -0,0 +1,75 @@
+(pd-dg-ml)=
+
+# Data Governance for the Machine Learning Pipeline
+
+A Machine Learning (ML) pipeline consists of a series of activities including the collection of data, training of an ML model, and the deployment of the model into use. 
+Data is integral throughout the ML process and the methods for which data is collected, annotated, processed, and shared will impact individuals and communities who may be represented in or the creators of the data, as well as data users who would like access to the data. 
+
+This chapter will cover examples of data governance practices for ML for different steps in the pipeline, which may include but not be exclusive to:
+- Data Collection
+- Data Management
+- Data Processing
+
+(pd-dg-ml-dc)=
+## Data Collection
+
+Many ML models are trained using datasets collected by a research team, which may be proprietary, or by using an open dataset that is available for download (sometimes with restrictions on its usage, such as only being available for use in academic settings).
+The deep learning (DL) family of models, in particular, relies on massive corpuses of data such as text, code, images, sound, and other media.
+The process of data collection depends on the type and volume of data required and sources for acquisition.
+For example, a project that uses patient health record data versus a project that uses a dataset of millions of social media posts will require different processes for gaining access to data as well as actually collecting and storing the data.
+Many DL models rely on data scraped from the internet due to the sheer volume of digital content that is available on the web. 
+For example, [ImageNet datasets](https://www.image-net.org/about.php) are sourced from web images from image hosting websites like Flickr, and [LAION datasets](https://laion.ai/faq/) come from web crawling sources like Common Crawl.
+These methods of data collection through web scraping have raised issues regarding data quality and bias due to the nature of using uncurated Internet data, as well as issues of data rights as web scraped data sets often include copyrighted data or data obtained without consent that may be in violation of the data license and terms of use.
+To address issues related to respecting data licenses, some ML teams are choosing to only use open source or permissively licensed data.
```

Comment:
Even among open source data, there are incompatible licenses, I'm not sure how much needs to be covered here, maybe just:

```suggestion
To address issues related to respecting data licenses, ML teams should be mindful of using data only when compatible with the license, including open source or permissively licensed data.
```